### PR TITLE
Templates account

### PR DESCRIPTION
**Pull request by teleyinex**

### CODE REVIEW

**Comment by therealmarv** [comment]
:cake:

File: templates/account/update.html [hunk 1]
```diff
@@ -2,38 +2,98 @@
...
+            {{ upload_form.y2 }}
+            <img id="uploadPreview">
+            <div class="form-actions">
+                <button type="submit" name='btn' value="Upload" class="btn btn-primary btn-large" />{{ _('Upload') }}</button>
+            </div>
+            </fieldset>
+        </form>
+        <form method="post" class="form-horizontal" action="{{ url_for('account.update_profile', name=current_user.name) }}">
             <fieldset>
+                <h2 {% if form.errors %}name="profile"{%endif%}>Your profile</h2>
```
Comment (alejandrodob) [inline templates/account/update.html]: gettext string

File: templates/account/update.html [hunk 2]
```diff
@@ -2,38 +2,98 @@
...
+                    <button type="submit" name="btn" value="Password" class="btn btn-primary btn-large">{{ _('Save the Changes')}}</button>
+                    <a href="{{ url_for('account.update_profile', name=current_user.name) }}" class="btn">{{ _('Cancel') }}</a>
+                </div>
+            </fieldset>
+        </form>
+        {% endif %}
+
+        <form method="post" class="form-horizontal" action="{{ url_for('account.update_profile', name=current_user.name) }}">
+        <hr>
+            <h2>External Services</h2>
```
Comment (alejandrodob) [inline templates/account/update.html]: gettext

**Comment by therealmarv** [comment]
I tested several pages. Found no errors so far. :cake:

**Pull request merged**